homeassistant/components/zwave_js/light.py

```diff
@@ -206,12 +206,23 @@ async def async_turn_on(self, **kwargs: Any) -> None:
         if white_value is not None and self._supports_white_value:
             # white led brightness is controlled by white level
             # rgb leds (if any) can be on at the same time
-            await self._async_set_colors(
-                {
-                    ColorComponent.WARM_WHITE: white_value,
-                    ColorComponent.COLD_WHITE: white_value,
-                }
-            )
+            white_channel = {}
+
+            if self.get_zwave_value(
```

Comment:
Let's cache the warm and cold target values as instance attributes in init method. Then we can use them here and don't need to fetch them again every time.